packages/forms-web-app/src/routes/index.js

```diff
@@ -7,6 +7,7 @@ const fullAppealAppellantSubmissionRouter = require('./full-appeal/submit-appeal
 const eligibilityRouter = require('./eligibility');
 const homeRouter = require('./home');
 const fullAppealRouter = require('./full-appeal');
+const householderPlanningEligibilityRouter = require('./householder-planning/eligibility');
```

Comment:
There should be an additional index in householder planning to pick up it's subsidiary routes. See my listed building ticket.

Reply:
Updated